src/test/tests/goUtils.test.ts

```diff
@@ -98,7 +98,7 @@ describe('Go Utils Tests', async () => {
         let textDocument: vscode.TextDocument = await vscode.workspace.openTextDocument(goMod);
 
         let dependenciesTreeNode: DependenciesTreeNode = new DependenciesTreeNode(
-            new GeneralInfo('github.com/jfrog/jfrog-cli-core', '1.9.1', [], '', '')
+            new GeneralInfo('github.com/jfrog/jfrog-cli-core', '1.9.1', [], '', PackageType.Unknown)
```

Comment:
Shouldn't be `PackageType.Go`?
Same question for all other appearances in this file and in the other similar tests